src/Microsoft.Kiota.Http.HttpClientLibrary.csproj

```diff
@@ -7,15 +7,15 @@
     <AssemblyTitle>Kiota Http Library for dotnet</AssemblyTitle>
     <Authors>Microsoft</Authors>
     <!-- NET 5 target to be removed on next major version-->
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0;net8.0;net462</TargetFrameworks>
```

Comment:
what's the rationale behind adding this additional target?

Reply:
Consumers running netFx would get their own dll as the code written is determined at compile time. This allows us to specify the required dependencies without affecting other targets as well as use the `NETFRAMEWORK` directive.

It's a recommended practice at 
https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting

<img width="651" alt="image" src="https://github.com/microsoft/kiota-http-dotnet/assets/6464005/5ebec04f-9a10-4a31-8dfb-73c2c4d3edc5">